extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ClassConfigPropertiesUtil.java

```diff
@@ -316,6 +321,10 @@ private static ResultHandle populateConfigObject(ClassLoader classLoader, ClassI
         return configObject;
     }
 
+    private static String getPropertyName(FieldInfo field, ConfigProperties.NamingStrategy namingStrategy) {
```

Comment:
Is this method still needed? I think you can call `namingStrategy.getName(field.name())` directly, no?

Reply:
Yeah, you are right. should have got rid of it, but I got mixed up because of the similar (but not exactly the same) case for interfaces

Reply:
So now it should be properly done :)